include/internal/numbers.h

```diff
@@ -61,6 +61,31 @@
 #  define UINT64_MAX __MAXUINT__(uint64_t)
 # endif
 
+/*
+ * 64-bit processor with LP64 ABI
+ */
+# ifdef SIXTY_FOUR_BIT_LONG
+#  ifndef UINT32_C
+#   define UINT32_C(c) (c)
+#  endif
+#  ifndef UINT64_C
```

Comment:
I actually wonder whether the one place that we use this, it is actually needed. The value passed can only be represented as an unsigned 64 bit int.

Reply:
Yes?  But depending on platform, it needs either UL or ULL at the end.

Or oh, you mean UINT32_C?